radio/src/storage/yaml/yaml_datastructs_funcs.cpp

```diff
@@ -114,7 +114,13 @@ static inline void check_yaml_funcs()
 {
   static_assert(offsetof(ModuleData, ppm) == 4,"");
   check_size<ModuleData, 29>();
+
+#if defined(PCBHORUS) && defined(PCBX10)
   static_assert(MAX_GVARS == 16,"");
+#else
+  static_assert(MAX_GVARS == 9,"");
+#endif
```

Comment:
The proper way here is to check which of the functions reused from the conversion uses the old constant, and copy/adapt the function here and remove the check.